test/core/end2end/end2end_chaotic_good_config.cc

```diff
@@ -124,32 +127,52 @@ class ChaoticGoodFixture : public InsecureFixture {
   std::string localaddr_;
 };
 
-class ChaoticGoodSingleConnectionFixture final : public ChaoticGoodFixture {
+class ChaoticGoodSingleConnectionFixture final
+    : public ChaoticGoodFixture<InsecureFixture> {
  public:
-  ChaoticGoodSingleConnectionFixture() : ChaoticGoodFixture(1) {}
+  ChaoticGoodSingleConnectionFixture() : ChaoticGoodFixture(0) {}
```

Comment:
Why did this parameter change from 0 to 1?

Reply:
It should have been to start with... and i need to test this case now.

chaotic-good defaults to it being 1 (number of data connections) -- because tradition
we need to additionally test the 0 case -- no data connections, everything over control